repos/fdbt-site/src/pages/products/services.tsx

```diff
@@ -85,18 +97,30 @@ const ServicesTable = (services: MyFaresServiceWithProductCount[]): ReactElement
     );
 };
 
-export const getTag = (startDate: string, endDate: string | undefined, isWithinATable: boolean): JSX.Element => {
+export const getEntityStatus = (startDate: string, endDate: string | undefined): EntityStatus => {
```

Comment:
What was the reason we are splitting this into 2 functions?

Reply:
`getTag` did two things. It worked out the status **and** returned the HTML markup as a `JSX.Element`.

I split out the logic for working out the status, then re-used that within the `getTag`. This allowed me to re-use the `getEntityStatus` function elsewhere, in place that where I wanted to know the status, but didn't need the actual HTML tag.